superset-frontend/src/components/Datasource/DatasourceEditor.jsx

```diff
@@ -1003,6 +1004,15 @@ class DatasourceEditor extends React.PureComponent {
           )}
           control={<CheckboxControl controlId="normalize_columns" />}
         />
+        <Field
+          inline
+          fieldKey="time_secondary_column"
+          label={t('Time secondary column')}
+          description={t(
+            `When the secondary datetime columns are filtered, apply the same filter to the main datetime column.`,
```

Comment:
Nit: we call them "temporal" columns, since technically they don't have to be a `datetime`.

```suggestion
            `When the secondary temporal columns are filtered, apply the same filter to the main temporal column.`,
```